pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -132,34 +155,35 @@ func (c *Cluster) Start() error {
 	}()
 
 	if c.Storage.UseAllNodes {
-		// make a daemonset for all nodes in the cluster
```

Comment:
if the operator dies and restarts in the middle of provisioning or starting osds, can we expect the operation to resume successfully? (ie. is it idempotent?)